lms/djangoapps/certificates/signals.py

```diff
@@ -70,14 +70,16 @@ def _listen_for_track_change(sender, user, **kwargs):  # pylint: disable=unused-
     grade_factory = CourseGradeFactory()
     for enrollment in user_enrollments:
         if grade_factory.read(user=user, course=enrollment.course_overview).passed:
-            if fire_ungenerated_certificate_task(user, enrollment.course_id):
-                log.info(u'Certificate generation task initiated for {user} : {course} via track change'.format(
-                    user=user.id,
-                    course=enrollment.course_id
+            if fire_ungenerated_certificate_task(user, enrollment.course_id, status):
+                log.info(u'Certificate generation task initiated for {user} : {course} via track change ' +
+                         u'with verification status of {status}'.format(
+                            user=user.id,
+                            course=enrollment.course_id,
+                            verification_status=verification_status
                 ))
 
 
-def fire_ungenerated_certificate_task(user, course_key):
+def fire_ungenerated_certificate_task(user, course_key, verification_status):
```

Comment:
Make this default to `None`, since the other two signal handler functions won't pass in anything.

Reply:
(also change name to `expected_verification_status`)